app/models/line_item.rb

```diff
@@ -0,0 +1,5 @@
+class LineItem < ApplicationRecord
+	belongs_to :item
+	belongs_to :order_item
+	validates  :quantity, :numaricality => {greater_than:0}
```

Comment:
use new hash syntax
numericality: { greater_than: 0 }

Reply:
add rubocop

Reply:
done it